flow-typed/classes/track/track.js

```diff
@@ -0,0 +1,34 @@
+//@flow
+export default class Track {
```

Comment:
flow-typed is for flow declarations only, e.g. class or interface definitions for example.
Actual concrete classes need to be part of the src/lib directory